pynta/main.py

```diff
@@ -440,24 +450,46 @@ def setup_transition_states(self,adsorbates_finished=False):
         Sets up fireworks to generate and filter a set of TS estimates, optimize each unique TS estimate,
         and run vibrational and IRC calculations on the each unique final transition state
         Note the vibrational and IRC calculations are launched at the same time
+
+        If irc_mode is "fixed", entire slab layers are fixed. 
```

Comment:
This doesn't make sense as a part of this docstring, irc_mode isn't a parameter of this function so this isn't a good location. We should probably add irc_mode to the documentation as a part of this PR though.

Reply:
Added irc_mode key to README.rst in docsrc